### ISSUE TITLE
It's difficult to see the branch breaks in `gh pr --detailed` view.  

### ISSUE THREAD

**Issue by drewbrokke** [node-gh/gh#580]
When viewing PRs across multiple branches, the standard `gh pr` is not so bad:
<img width="758" alt="screen shot 2018-10-17 at 11 11 20 am" src="https://user-images.githubusercontent.com/6403097/47107158-6b2f6780-d1fd-11e8-8806-62419e38d524.png">
But when calling `gh pr --detailed`, the visual noise is hard to parse through:
<img width="835" alt="screen shot 2018-10-17 at 11 12 26 am" src="https://user-images.githubusercontent.com/6403097/47107231-9d40c980-d1fd-11e8-9a00-2eeedeb69573.png">
My 2 gripes are: 
1. It's not obvious when a new pull begins besides the green number.  Perhaps we could bold the title and/or introduce a line break after each pull summary?
2. The branch listings are too understated.  Same suggestion as above :)

**Comment by protoEvangelion** [comment]
I agree! We cleaned up the formatting for listing issues and this could use a revamp too!

**Comment by protoEvangelion** [comment]
If someone wants to take a shot at this, you will want to start here: https://github.com/node-gh/gh/blob/master/lib/cmds/pull-request.js#L461

**Comment by protoEvangelion** [comment]
Related: #611, #595, #589

**Comment by protoEvangelion** [comment]
@drewbrokke here is what it looks like now:

![image](https://user-images.githubusercontent.com/20076677/52315905-d9936380-296d-11e9-8b8b-f1fb5114dd5a.png)

Is that good enough for you?

**Comment by drewbrokke** [comment]
@protoEvangelion that looks really nice. Do you have an example of what it looks like when viewing pulls across multiple repos?  That was my original gripe :)

**Comment by protoEvangelion** [comment]
It starts a new table with new headers so it is pretty clear visually

![image](https://user-images.githubusercontent.com/20076677/52316390-faf54f00-296f-11e9-9593-5701d87f58b0.png)

**Comment by drewbrokke** [comment]
PR send to https://github.com/node-gh/gh/pull/631

**Comment by drewbrokke** [comment]
Resent as https://github.com/node-gh/gh/pull/632
